Statistics/Distributions/LogNormal.cs

```diff
@@ -35,13 +35,33 @@ public LogNormal()
         /// <param name="mean"></param> mean of the logged data
         /// <param name="sd"></param> standard deviation of the logged data
         /// <param name="sampleSize"></param>
-        public LogNormal(double mean, double sd, int sampleSize = int.MaxValue)
+        ///  <param name="unlogged"></param> Unlogged is true if the mean and standard deviation reflect unlogged data 
+        public LogNormal(double mean, double sd, int sampleSize = int.MaxValue, bool unlogged = false)
         {
-            Mean = mean;
-            StandardDeviation = sd;
+            if (unlogged == false)
```

Comment:
do we really need this feature? will it ever be used by our code in fda?

Reply:
I was attempting to make better use of Table 6-3 in EM 1110-2-1619 according to the footnote in that table, but was unsuccessful, you'll see I later deleted that line.